env/env_posix.cc

```diff
@@ -302,19 +304,22 @@ class PosixEnv : public CompositeEnv {
     return thread_status_updater_->GetThreadList(thread_list);
   }
 
-  static uint64_t gettid(pthread_t tid) {
-    uint64_t thread_id = 0;
+  uint64_t GetThreadID() const override {
+    uint64_t thread_id;
```

Comment:
Nit: I think it's still a good idea to initialize to 0.

Reply:
Done.